unexpand/src/unexpand.yml

```diff
@@ -0,0 +1,24 @@
+name: unexpand
+version: "0.1"
+author: "Bojan Durdevic <bojan88@gmail.com>"
+about: Convert spaces to tabs
+args:
+    - all:
+        help: convert all blanks, instead of just initial blanks
+        short: a
+        long: all
+    - first_only:
```

Comment:
All options/flags should have a short and a long one, it it doesn't have on the original, you are free to create one